libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/RouterWrapper.kt

```diff
@@ -230,15 +239,18 @@ class RouterWrapper(
                                code = ${it.code}
                                type = ${it.type}
                                requestId = ${it.requestId}
+                               refreshTTL = ${it.refreshTtl}
                                routeRefreshRequestData = $routeRefreshRequestData
                             """.trimIndent()
 
                         logW(errorMessage, LOG_CATEGORY)
 
+                        RouteExpirationHandler.updateRouteExpirationData(route, it.refreshTtl)
```

Comment:
Why does error change refresh ttl?

Reply:
It can in case there was an untimely route invalidation.

Reply:
Does it mean that `it.refreshTtl` is always 0? 🤔

Reply:
At the moment server only plans to send errors with refresh_ttl = 0. However, the API does not prohibit from sending anything else.